templates/web/base/waste/garden/renew.html

```diff
@@ -12,37 +12,48 @@
 <form method="post" id="renew">
     [% INCLUDE waste/garden/_bin_quantities.html form_page='renew' %]
   <hr class="fieldset-hr">
-    <div class="cost-pa__total js-bin-costs"
+  <div class="cost-pa__total js-bin-costs"
+    [% IF form.saved_data.apply_discount %]
+        data-per_bin_cost="[% c.cobrand.apply_garden_waste_discount(per_bin_cost) %]"
+        data-per_new_bin_first_cost="[% c.cobrand.apply_garden_waste_discount(per_new_bin_first_cost) %]"
+        data-per_new_bin_cost="[% c.cobrand.apply_garden_waste_discount(per_new_bin_cost) %]"
+    [% ELSE %]
         data-per_bin_cost="[% per_bin_cost %]"
         data-per_new_bin_first_cost="[% per_new_bin_first_cost %]"
         data-per_new_bin_cost="[% per_new_bin_cost %]"
-      [% IF is_staff && waste_features.ggw_discount_as_percent %]
-        data-discount_percent="[% waste_features.ggw_discount_as_percent %]"
-      [% END %]
-    >
-    [% IF is_staff && waste_features.ggw_discount_as_percent %]
-        <span hidden class="cost-pa__total-costs">
-            <span id="apply_discount_wrapper" hidden>Discount applied: <span id="apply_discount_amount">[% tprintf( '%.0f', waste_features.ggw_discount_as_percent ) %]%</span></span>
-        </span>
     [% END %]
+    >
+    [% IF form.saved_data.apply_discount %]
```

Comment:
Similarly here for cost_pa/cost_now_admin/cost_now - unlike the per_bin variables, these ones have already been discounted. Unless I've missed something?